final_project/UITestFixture.cs

```diff
@@ -17,21 +56,77 @@ public async Task Setup()
                 Headless = false
             });
 
+            string subPath = "../../../playwright/auth";
+            string filePath = "../../../playwright/auth/state.json";
+
+            if(!Directory.Exists(subPath))
+            {
+                Directory.CreateDirectory(subPath);
+            }
+
+            if(!File.Exists(filePath))
+            {
+                File.AppendAllText(filePath, "{}");
+            }
+
             context = await browser.NewContextAsync(new BrowserNewContextOptions
             {
                 ViewportSize = new ViewportSize
                 {
                     Width = 1920,
                     Height = 1080
-                }
+                },
```

Comment:
this is second context with the state (for UI tests), so it should be in a separate variable, but not to override context created in OneTimeSetUp